src/components/field-label/README.md

```diff
@@ -30,6 +30,23 @@ import { FieldLabel } from '@commercetools-frontend/ui-kit';
 />
 ```
 
+The `hintIcon` also accepts a custom `theme` while defaulting to `orange` in the case above.
+
+```diff
+<FieldLabel
+  title={<FormattedMessage {...messages.title} />}
+  hasRequiredIndicator={true}
+  onInfoButtonClick={() => {}} />}
+  hint={<FormattedMessage {...messages.hint} />}
+- hintIcon={<WarningIcon />}
++ hintIcon={<WarningIcon theme="green" />}
```

Comment:
Found the `diff` type of illustrate best what the difference is between both examples.